src/routes/zoom.routes.js

```diff
@@ -0,0 +1,217 @@
+        + '?response_type=code'
+        + '&client_id=' + ZOOM_CLIENT_ID
+        + '&redirect_uri=' + ZOOM_REDIRECT_URI;
+
+        return res.json({
+            msg: "200 Ok",
+            data: uri
+        })
+    } catch(e) {
+        return next(e);
+    }
+});
+
+// Procesado del código de zoom devuelto al landing page.
+// Devuelve un token de acceso y uno de refresco, almacenados en la BD referidos al usuario.
+router.post('/zoom/token', async (req, res, next) => {
+    try {
+
+        const respuesta = await got.post('https://zoom.us/oauth/token'
+        + '?code=' + req.body.code
+        + '&redirect_uri=' + ZOOM_REDIRECT_URI
+        + '&grant_type=authorization_code', {
+            headers: {
+                "Authorization": "Basic " + Buffer.from(ZOOM_CLIENT_ID + ':' + ZOOM_CLIENT_SECRET).toString('base64')
+            }
+        });
+
+        const datosParseados = JSON.parse(respuesta.body)
+
+        const { email } = req.body
+        const usuario = await Usuario.findOne({ email })
+
+        const preDatosZoom = await ZoomDatosUsuarios.findOne({ userId: mongoose.Types.ObjectId(usuario._id) });
+
+        // Si existen datos previos lanzamos un error
+        if(preDatosZoom) return next(new Error("El usuario ya cuenta con datos de acceso a Zoom"))
+
+        const zoomDatosUsuarios = new ZoomDatosUsuarios({
+            userId: usuario._id,
+            access_token: datosParseados.access_token,
+            refresh_token: datosParseados.refresh_token,
+            expires_in: datosParseados.expires_in
+        })
+
+        await zoomDatosUsuarios.save()   
+        
+        //Si existen datos previos se actualizan con los nuevos, si no se crean nuevos y se guardan
+        /*
+        if(preDatosZoom) {
+            preDatosZoom.userId = usuario._id;
+            preDatosZoom.access_token = datosParseados.access_token;
+            preDatosZoom.refresh_token = datosParseados.refresh_token;
+            preDatosZoom.expires_in = datosParseados.expires_in;
+
+            await preDatosZoom.save()      
+        } else {
+            const zoomDatosUsuarios = new ZoomDatosUsuarios({
+                userId: usuario._id,
+                access_token: datosParseados.access_token,
+                refresh_token: datosParseados.refresh_token,
+                expires_in: datosParseados.expires_in
+            })
+
+            await zoomDatosUsuarios.save()      
+        }
+        */
+
+        return res.json({
+            msg: "200 Ok",
+            data: JSON.parse(respuesta.body)
+        });
+    } catch(e) {
+        return next(e);
+    }
+});
+
+// Elimina los datos de Zoom (access_token, refresh_token, etc...) de un usuario por su email
+router.delete('/zoom/token', async (req, res, next) => {
+    try {
+        const { email } = req.body
+        const usuario = await Usuario.findOne({ email })
+        
+        if(!usuario) return next(new Error("No existen datos del usuario"))
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7157741)
```suggestion
        if(!usuario) {return next(new Error("No existen datos del usuario"))}
```